esp/esp/users/urls.py

```diff
@@ -25,6 +26,7 @@
     url(r'^disableaccount/?$', views.disable_account),
     url(r'^grade_change_request/?$', GradeChangeRequestView.as_view(), name = 'grade_change_request'),
     url(r'^makeadmin/?$', views.make_admin),
+    url(r'^loginhelp', TemplateView.as_view(template_name='users/loginhelp.html'), name='Login Help'),
```

Comment:
Do you mind putting this in a separate pull request?  It's hard to see in with all the other changes.

Reply:
If y'all promise to review that one quickly :)
Sincerely,
Pot Calling Kettle Black